eva_cttv_pipeline/trait_mapping/trait_names_parsing.py

```diff
@@ -28,8 +28,7 @@ def parse_trait_names(filepath: str) -> list:
     nt_expansion_traits = set()
 
     for clinvar_record in clinvar_xml_utils.ClinVarDataset(filepath):
-        trait_names = set(trait.preferred_or_other_name.lower() for trait in clinvar_record.traits
-                          if trait.preferred_or_other_name is not None)
+        trait_names = set(trait.preferred_or_other_valid_name.lower() for trait in clinvar_record.valid_traits)
```

Comment:
I think this should be:
```suggestion
        trait_names = set(trait.preferred_or_other_valid_name.lower() for trait in clinvar_record.traits_with_valid_names)
```

Reply:
You're absolutely right, thanks!